### PR TITLE
Traditional Chinese Translation

### PR DESCRIPTION
**Pull request by Heinrich-XIAO**

### CODE REVIEW

**Comment by Heinrich-XIAO** [comment]
Translaton done. The only real difference between traditional and simplified chinese is a difference in some of the characters.

**Comment by Naviary2** [comment]
There seems to be only 146 lines in the Traditional Chinese file, where I was expecting 600+ to match the others

File: translation/zh-TW.toml [hunk 1]
```diff
@@ -357,7 +357,7 @@ please_report_bug = "這不應該發生，請報告此錯誤！"
 checkmate = "將死"
 stalemate = "僵局"
 repetition = "三次重復"
-moverule = [, "-回合規則"]  # The game inserts a number inbetween these two strings
+moverule = ["", "-回合規則"]  # The game inserts a number inbetween these two strings
```
Comment (Naviary2) [inline translation/zh-TW.toml:360]: I meant to say missing quotes, not parenthesis

**Comment by Naviary2** [comment]
Thank you!

**Pull request merged**